os/user/getent_cgo_test.go

```diff
@@ -0,0 +1,30 @@
+// +build aix dragonfly freebsd !android,linux netbsd openbsd solaris darwin
+// +build cgo,!osusergo
+
+package user
+
+import (
+	"errors"
+	"testing"
+)
+
+// As we can't really test cgo, we will attempt to
+// check if user/group record could be at least retrieved
```

Comment:
can we use `fgetpwent` for tests, and pass a file fixture for it? Then we can assert both full records and error conditions.